src/main/scala/mesosphere/marathon/api/LeaderProxyFilter.scala

```diff
@@ -294,32 +292,29 @@ class JavaUrlConnectionRequestForwarder @Inject() (
             () => cloneResponseEntity(leaderConnection, response)
           )
         } finally {
-          Try(leaderConnection.getInputStream.close())
-          Try(leaderConnection.getErrorStream.close())
+          Closeables.closeQuietly(leaderConnection.getInputStream())
+          Closeables.closeQuietly(leaderConnection.getErrorStream())
         }
       }
     } finally {
-      Try(request.getInputStream.close())
-      Try(response.getOutputStream.close())
+      Closeables.closeQuietly(request.getInputStream())
+      Closeables.close(response.getOutputStream(), true)
```

Comment:
why not closeQuietly here? If I see it correctly in the code, it does the same (close quietly calls close with swallow exception)

Reply:
This is confusing, I know. There is no `closeQuietly` for output streams.

Reply:
oh I see... that's weird :) ok thanks for the explanation